src/buffer/out/textBuffer.hpp

```diff
@@ -311,10 +311,10 @@ class TextBuffer final
 
 private:
     void _reserve(til::size screenBufferSize, const TextAttribute& defaultAttributes);
-    void _commit(const std::byte* row);
-    void _decommit() noexcept;
-    void _construct(const std::byte* until) noexcept;
-    void _destroy() const noexcept;
+    void _commit(const uint8_t* row);

```

Comment:
hmm why these?

Reply:
This is so that `cls` releases the memory again. I figured that this would become much more important once the scrollback can actually become quite large.

Reply:
Oh, you meant `std::byte` vs. `uint8_t`. We only use the former in a teeny tiny subset of the codebase, of which this usage here is half of it. By using the more traditional `uint8_t` we make our code simpler and need less specialization. It's also more predictable, because primitive integers have received compiler optimizations over decades whereas `std::byte` hasn't.
I'm reminded of that one blog post that showed how poorly `std::byte` was optimized by compilers at the time, but I was unable to find it while writing this comment.